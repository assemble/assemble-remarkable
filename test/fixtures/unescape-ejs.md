# ejs templates

<%= foo %>
<%- foo %>
<% foo %>
<% _.forEach(users, function(user) { %><li><%- user %></li><% }); %>
