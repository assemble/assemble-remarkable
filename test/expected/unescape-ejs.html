<h1>ejs templates</h1>
<p><%= foo %>
<%- foo %>
<% foo %>
<% _.forEach(users, function(user) { %><li><%- user %></li><% }); %></p>
